homeUi/src/main/kotlin/com/gravatar/app/homeUi/presentation/home/share/components/ItemDivider.kt

```diff
@@ -0,0 +1,18 @@
+package com.gravatar.app.homeUi.presentation.home.share.components
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+
+@Composable
+internal fun ItemDivider(modifier: Modifier = Modifier) {
+    Column(modifier) {
+        Spacer(modifier = Modifier.height(8.dp))
+        HorizontalDivider(thickness = 1.dp)
```

Comment:
⛏️  The color in the designs looks lighter.

```suggestion
        HorizontalDivider(thickness = 1.dp, color = DividerDefaults.color.copy(alpha = 0.3f))
```

Reply:
[e4a12b2](https://github.com/Automattic/Gravatar-Android/pull/77/commits/e4a12b28e7f9e77e10e3fb7a6f6cb4c080258166)